pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1610,3 +1633,15 @@ func admitMigrationUpdate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRespons
 func ServeMigrationUpdate(resp http.ResponseWriter, req *http.Request) {
 	serve(resp, req, admitMigrationUpdate)
 }
+
+func ValidateDuplicateDHCPPrivateOptions(PrivateOptions []v1.DHCPPrivateOptions) error {
+	isUnique := map[int]bool{}
+	for _, index := range PrivateOptions {
```

Comment:
this is not the index it is a DHCPPrivateOption object maybe change the name?

Reply:
changed this to DHCPPrivateOption! :)